redshift.tf

```diff
@@ -133,6 +133,7 @@ resource "aws_redshift_cluster" "redshift_cluster" {
   provisioner "local-exec" {
```

Comment:
This does not need to be here anymore as the tables are created via the startup script for the EC2 instance.

Reply:
Thanks for calling that out. The lcoal -exec block is now deleted.